website/docs/dev_launcher.md

```diff
@@ -107,6 +111,23 @@ Environment variables that are set for backwards compatibility with openpype add
 - **OPENPYPE_REPOS_ROOT** - Alias to **AYON_ROOT**.
 - **AVALON_LABEL** - Alias to **AYON_MENU_LABEL**.
 
+:::note
+Environment variables **AYON_LAUNCHER_STORAGE_DIR** and **AYON_LAUNCHER_LOCAL_DIR** are by default set to the same folder. Path is based on OS.
+- Windows: `%LOCALAPPDATA%\Ynput\AYON`
+- Linux: `~/.local/share/AYON`
+- macOS: `~/Library/Application Support/AYON`
+
+It is required to set the environment variables before AYON launcher is started as it is required for bootstrap.
+:::
+
+:::tip
+Environment variables **AYON_ADDONS_DIR** and **AYON_DEPENDENCIES_DIR** by default are relative to **AYON_LAUNCHER_STORAGE_DIR**.
+
+- **AYON_ADDONS_DIR** -> `{AYON_LAUNCHER_STORAGE_DIR}/addons`
+- **AYON_DEPENDENCIES_DIR** -> `{AYON_LAUNCHER_STORAGE_DIR}/dependency_packages`
+
+Changing their values will change where addons and dependency packages are stored even if you change **AYON_LAUNCHER_STORAGE_DIR**!
+:::
```

Comment:
what about ? 
```suggestion
:::info Launcher Storage Environments
Before launching the AYON launcher, it's important to set up the necessary environment variables for the initial bootstrapping process.

Here are the default paths for the storage environment variables, which vary based on your operating system:
- **AYON_LAUNCHER_STORAGE_DIR** and **AYON_LAUNCHER_LOCAL_DIR**: Both default to the following paths depending on your OS:
    - Windows: `%LOCALAPPDATA%\Ynput\AYON`
    - Linux: `~/.local/share/AYON`
    - macOS: `~/Library/Application Support/AYON`

Additionally, the following directories are set by default and are relative to **AYON_LAUNCHER_STORAGE_DIR**:
- **AYON_ADDONS_DIR**: `{AYON_LAUNCHER_STORAGE_DIR}/addons`
- **AYON_DEPENDENCIES_DIR**: `{AYON_LAUNCHER_STORAGE_DIR}/dependency_packages`

Please note that by default, **AYON_ADDONS_DIR** and **AYON_DEPENDENCIES_DIR** are set relative to **AYON_LAUNCHER_STORAGE_DIR**. However, if you choose to modify these variables, they will no longer reference the default storage directory.
:::
```
Here's how it looks on my side:
![image](https://github.com/user-attachments/assets/754e5c13-54f2-423a-9c55-e3d4fa53dacd)

Reply:
I would say this is for technical people, more words make it more complicated and less understandable.